shared/engine/index.js

```diff
@@ -206,23 +224,27 @@ class Engine {
 
     const callMap = this.sessionIDToIncomingCall[sessionID]
 
+    if (printRPC) {
+      console.log('RPC ◀ incoming: ', payload)
+    }
+    // make wrapper so we only call this once
+    const wrappedResponse = this._wrapResponseOnceOnly(method, param, response)
+
     if (callMap && callMap[method]) {
-      // make wrapper so we only call this once
-      const wrappedResponse = this._wrapResponseOnceOnly(method, param, response)
       callMap[method](param, wrappedResponse)
     } else if (method === 'keybase.1.logUi.log' && this._hasNoHandler(method, callMap || {}, this._generalIncomingRpc)) {
       log(param)
-      response.result()
+      wrappedResponse.result()
```

Comment:
let's always call the wrapped response, we only want to respond once and we should collapse the different logical flows